var/spack/repos/builtin/packages/axom/package.py

```diff
@@ -93,6 +93,7 @@ class Axom(CachedCMakePackage, CudaPackage, ROCmPackage):
     # Basics
     depends_on("cmake@3.8.2:", type="build")
     depends_on("cmake@3.16.8:", type="build", when="+rocm")
+    depends_on("cmake@3.21:", type="build", when="@0.7.0")
```

Comment:
This is only true `when="+rocm"`:

https://github.com/LLNL/axom/blob/e605cfcb1fce1e1a58703dbc95f4eb9b95329f99/src/CMakeLists.txt#L9-L13

Otherwise its 3.14+

Reply:
Just committed the change you suggested in other post. Thank you!